logo/generate_mne_logos.py

```diff
@@ -126,26 +132,36 @@
     - tagline_offset_fudge
 )
 tag_clip = Path(offset + vert * mult, tag_path.codes)
-tag_patch = PathPatch(tag_clip, facecolor="k", edgecolor="none", zorder=10)
+tag_patch = PathPatch(tag_clip, facecolor="0.6", edgecolor="none", zorder=10)
 ax.add_patch(tag_patch)
 yl = ax.get_ylim()
 yy = np.max([tag_clip.vertices.max(0)[-1], tag_clip.vertices.min(0)[-1]])
 ax.set_ylim(np.ceil(yy), yl[-1])
 
 # only save actual image extent plus a bit of padding
-plt.draw()
-static_dir = op.join(op.dirname(__file__), "..", "doc", "_static")
-assert op.isdir(static_dir)
-plt.savefig(op.join(static_dir, "mne_logo.svg"), transparent=True)
-tag_patch.set_facecolor("w")
-rect.set_facecolor("0.5")
-plt.savefig(op.join(static_dir, "mne_logo_dark.svg"), transparent=True)
-tag_patch.set_facecolor("k")
-rect.set_facecolor("w")
+fig.canvas.draw_idle()
+static_dir = pathlib.Path(__file__).parent / ".." / "doc" / "_static"
```

Comment:
[nitpick] seems weird to mix `.parent / ".."` right after one another

```suggestion
static_dir = pathlib.Path(__file__).parent.parent / "doc" / "_static"
```

Reply:
why not use `.parents[1]`

Reply:
that will only work if you `resolve()`

Reply:
actually never mind, `Path(__file__)` should be absolute, so the `resolve()` isn't necessary